src/board.py

```diff
@@ -3,8 +3,9 @@
 Referencing with chess algebraic notation is possible.
 """
 
-from .pieces import Bishop, Color, King, Knight, Pawn, Piece, Queen, Rook
-from .square import Square, Vector
+from typing import Callable
+from src.pieces import Bishop, Color, King, Knight, Pawn, Piece, Queen, Rook
+from src.square import Square, Vector
```

Comment:
Booohooo, my relative imports, sniff.

Reply:
For some reason, relative imports do not work for me. It only works like this (`from piece`, `from square`) etc.